### ISSUE TITLE
bug: deleting plugin doesn't take effect

### ISSUE THREAD

**Issue by lxz999** [apache/apisix#10085]
### Current Behavior

After the ip blacklist and whitelist plug-in is deleted in apisxi3.0, the restriction still exists by ip. The restriction takes effect only after apisix is restarted

### Steps to Reproduce

1. Add the ip-restriction plug-in to the service and add whitelist restrictions for the plug-in to take effect
2. After the IP-restriction plug-in is deleted, the related interface is still restricted by ip

### Environment

- APISIX version (run `apisix version`):
- Operating system (run `uname -a`):
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):
APISIX 3.0版本

**Comment by Revolyssup** [comment]
Did you reload with
`curl http://127.0.0.1:9180/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT` ?

**Comment by shreemaan-abhishek** [comment]
> plug-in is deleted in apisxi3.0

What did you do to delete the plugin?

**Comment by lxz999** [comment]
> Did you reload with `curl http://127.0.0.1:9180/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT` ?

I tried. I can't

> > plug-in is deleted in apisxi3.0
> 
> What did you do to delete the plugin?
Call admin api to delete

**Comment by Revolyssup** [comment]
> 2\. still restricted by ip

Can you paste the exact request you sent to delete?

**Comment by lxz999** [comment]
> Did you reload with `curl http://127.0.0.1:9180/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT` ?

I've tried, but I can't. I see that the apisix log does not reload the plugin

> > 2. still restricted by ip
> 
> Can you paste the exact request you sent to delete?

This problem occurred when I called the admin api too often, and then calling reload didn't work either, and there was no log output in the background. Are there any restrictions on calling the admin api

> > 2. still restricted by ip
> 
> Can you paste the exact request you sent to delete?

This problem occurred when I called the admin api too often, and then calling reload didn't work either, and there was no log output in the background. Are there any restrictions on calling the admin api
